src/Sylius/Behat/Context/Ui/Admin/ManagingShipmentsContext.php

```diff
@@ -58,4 +58,21 @@ public function iShouldSeeCountShipmentsInList(int $count): void
     {
         Assert::same($count, $this->indexPage->countItems());
     }
+
+    /**
+     * @Then I ship shipment order :orderNumber
+     */
+    public function iShipShipmentOrder(string $orderNumber): void
+    {
+        $this->indexPage->shipShipmentWithSpecificOrderNumber($orderNumber);
+    }
+
+    /**
+     * @Given I should see order :orderNumber as :shippingState
+     */
+    public function iShouldSeeOrderAs(string $orderNumber, string $shippingState): void
+    {
+        $order = $this->indexPage->getRowWithSpecificOrderNumber($orderNumber);
+        Assert::same($shippingState, $order);
```

Comment:
`Assert::same($shippingState, $this->indexPage->getRowWithSpecificOrderNumber($orderNumber));` would fit